ext_conf_template.txt

```diff
@@ -1,2 +1,5 @@
 # cat=basic; type=boolean; label=Enable RTE in replacement field. Experimental!
 enableRte = 0
+
+# cat=basic; type=boolean; label=Remove empty markers if still there
+removeAllMarkers = 0
```

Comment:
Functionality is fine, but I'm not sure about the naming here. Wouldn't sth like `removeUnreplacedMarkers` be more speaking?

Reply:
Merci